builder/docker/tinygo/smoke.sh

```diff
@@ -44,7 +45,7 @@ if [ $? -ne 0 ]; then
 fi
 
 # Check the base image
-assert_file $base/Dockerfile.base $version 'branch \K.+(?= https://github.com/tinygo-org/tinygo.git)'
+assert_file $base/Dockerfile.base $base_version 'branch \K.+(?= https://github.com/tinygo-org/tinygo.git)'
```

Comment:
Updated this line @jagger27 as it is impossible for both the base image and builder image to be updated at the same time.

Instead, it uses `yq` to ensure that the `cloudbuild.yaml` version and the clone are the same.